models/migrations/migrations.go

```diff
@@ -80,7 +80,9 @@ var migrations = []Migration{
 	NewMigration("create user column diff view style", createUserColumnDiffViewStyle),
 	// v15 -> v16
 	NewMigration("create user column allow create organization", createAllowCreateOrganizationColumn),
-	// v16
+	// V16 -> v17
+	NewMigration("create repo unit table and add units for all repos", addUnitsToTables),
```

Comment:
there was a new one added in master, mine  is after this one
https://github.com/go-gitea/gitea/pull/679/files#diff-7366023de6e87b891be868e3b8cf68e6R85-R86